tests/layer_tests/tensorflow_tests/test_tf_Conv2D.py

```diff
@@ -0,0 +1,82 @@
+    # input_strides - should be an array, defines strides of a sliding window to use
+    # input_padding - should be a string, defines padding algorithm
+    # ir_version - common parameter
+    # use_new_frontend - common parameter
+    def create_conv2d_placeholder_const_net(self, input_shape, input_filter, input_strides, input_padding, dilations, ir_version, use_new_frontend):
+        """
+            Tensorflow net                  IR net
+
+            Placeholder->Conv2D    =>       Placeholder->Transpose->Convolution--------------------->Reshape
+                                                                               \                   /
+                                                                                \-ShapeOf->Gather-/
+
+        """
+
+        import tensorflow as tf
+
+        if dilations is None:
+            dilations = [1, 1, 1, 1] #default value regarding Documentation
+
+        #               Batch   Height Width  Channel
+        expl_paddings = [0, 0,  1, 1,  1, 1,  0, 0]
+
+        if input_padding == 'EXPLICIT' and use_new_frontend == False:
+            pytest.xfail(reason="100300")
+
+        tf.compat.v1.reset_default_graph()
+
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            tf_input = tf.compat.v1.placeholder(tf.float32, input_shape, "InputShape")
+            tf_filter = tf.compat.v1.placeholder(tf.float32, input_filter, "InputFilter")
+
+            if input_padding != 'EXPLICIT':
+                tf.raw_ops.Conv2D(input = tf_input, filter = tf_filter, strides = input_strides, padding = input_padding, dilations = dilations)
+            else:
+                tf.raw_ops.Conv2D(input = tf_input, filter = tf_filter, strides = input_strides, padding = input_padding, explicit_paddings=expl_paddings, dilations = dilations)
+
+            tf.compat.v1.global_variables_initializer()
+            tf_net = sess.graph_def
+
+        ref_net = None
+
+        return tf_net, ref_net
+
+    test_data = [
+        dict(input_shape=[1, 10, 10, 1], input_filter=[3, 3, 1, 1], input_strides=[1, 1, 1, 1], dilations=None),
+        dict(input_shape=[1, 10, 10, 2], input_filter=[3, 3, 2, 2], input_strides=[1, 1, 1, 1], dilations=None),
+        dict(input_shape=[1, 10, 10, 2], input_filter=[3, 3, 2, 1], input_strides=[1, 1, 1, 1], dilations=None),
+        dict(input_shape=[1, 10, 10, 3], input_filter=[2, 2, 3, 3], input_strides=[1, 1, 1, 1], dilations=None),
+        dict(input_shape=[1, 16, 16, 3], input_filter=[2, 2, 3, 3], input_strides=[1, 2, 2, 1], dilations=None),
+        dict(input_shape=[1, 10, 10, 4], input_filter=[2, 2, 4, 2], input_strides=[1, 1, 1, 1], dilations=None),
+        dict(input_shape=[1, 16, 16, 3], input_filter=[2, 2, 3, 3], input_strides=[1, 2, 2, 1], dilations=[1, 2, 2, 1]),
+        pytest.param(
+            dict(input_shape=[1, 224, 224, 3], input_filter=[16, 16, 3, 1], input_strides=[1, 2, 2, 1], dilations=[1, 2, 2, 1]),
```

Comment:
`input_filter=[16, 16, 3, 1]` is really huge kernel, not practical, and may consume solid testing time. Please replace with the other filter as `input_filter=[4, 4, 3, 2]`.

Reply:
done